### ISSUE TITLE
Add Functionality

### ISSUE THREAD

**Issue by pingvikin** [DivineGenesis/BetterSoulBinding#9]
Can you add functionality so that a linked item appears in the same slot the player before his death after the player is revived?

Before death:
![image](https://user-images.githubusercontent.com/26678843/41439380-34ec79fa-7033-11e8-871b-97b8f2918d4f.png)

After death:
![image](https://user-images.githubusercontent.com/26678843/41439405-4c193fbe-7033-11e8-8be0-5a289c77ee75.png)

**Comment by burpingdog1** [comment]
Yeah. I do indeed agree this is important. The current system of keeping soulbinded items with you after death doesn't allow us to detect where the item is in the inventory pre-death. this might take awhile to add in BUT I will try my hardest to do this :) Wanted to post this just so you knew what was happening and that we weren't ignoring this. Thanks for bringing this idea up to Dino and I!

**Comment by pingvikin** [comment]
Thank you very much :) You are quite gallant
